pulp_rpm/app/serializers/advisory.py

```diff
@@ -231,6 +232,16 @@ def validate(self, data):
         validated_data = super().validate(update_record_data)
         return validated_data
 
+    # pulpcore 3.22 feature
+    # TODO upload has to be able to "find" a unique advisory, and id is Not It?
```

Comment:
Is this TODO a blocker?

Reply:
This is a problem for advisory-upload, and has been since we implemented the "if it already exists just hand the existing entity back". Advisories can only be truly unique-ified based on their digest, which isn't something you know until *after* it's been uploaded. It's not a domain-issue, it's an advisory-upload issue.